leaper.py

```diff
@@ -347,15 +347,6 @@ def check_source_submission(self, src_project, src_package, src_rev, target_proj
                 self.do_check_maintainer_review = False
                 return True
 
-            # new package submitted from Factory. Check if it was in
-            # 42.3 before and skip maintainer review if so.
-            subprj = src_project[len('openSUSE:Factory'):]
-            if self.source_in_factory and target_project.startswith('openSUSE:Leap:15.0') \
-                and self.is_package_in_project('openSUSE:Leap:42.3'+subprj, package):
-                    self.logger.info('package was in 42.3')
-                    self.do_check_maintainer_review = False
-                    return True
-
```

Comment:
Is this not something that will be desired to have in the future? If so I'd rather disable it like other conditions as it will make it a lot better if/when we decide to refactor this whole thing to know what the desired features actually are.